Makefile

```diff
@@ -1,10 +1,11 @@
 # Configuration
 	# -------------
 
-APP_NAME ?= `grep 'app:' mix.exs | sed -e 's/\[//g' -e 's/ //g' -e 's/app://' -e 's/[:,]//g'`
+APP_NAME := $(shell grep 'app:' mix.exs | sed -e 's/\[//g' -e 's/ //g' -e 's/app://' -e 's/[:,]//g')
 APP_VERSION := $(shell git fetch && git describe --tags `git rev-list --tags --max-count=1`)
 DOCKER_IMAGE_TAG ?= $(APP_VERSION)
 GIT_REVISION ?= `git rev-parse HEAD`
+SBOM_FILE_NAME_CY ?= $(APP_NAME).$(APP_VERSION).cyclonedx-sbom.1.0.0
```

Comment:
All sbom_file_name will end with `.cyclonedx-sbom.1.0.0 `.Looking at issue #367 spdx bom files should end with this format i.e ` .spdx-sbom.1.0.0.spdx` .As  per now  we haven't yet implemented on generation of spdx bom files. Will that be considered as future change?

Reply:
The variable name ends with `CY` to indicate that its a cyclone dx bom file. Once we have spdx on we will create a variable ending with `SP`.